src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/Services/ValidationOverrides/ValidationOverrideService.cs

```diff
@@ -0,0 +1,89 @@
+﻿using SFA.DAS.EmployerIncentives.Functions.LegalEntities.Services.ValidationOverrides.Types;
+using SFA.DAS.EmployerIncentives.Functions.LegalEntities.Services.Withdrawals.Types;
+using SFA.DAS.EmployerIncentives.Types;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EmployerIncentives.Functions.LegalEntities.Services.ValidationOverrides
+{
+    public class ValidationOverrideService : IValidationOverrideService
+    {
+        private readonly HttpClient _client;
+
+        public ValidationOverrideService(HttpClient client)
+        {
+            _client = client;
+        }
+
+        public async Task Add(IEnumerable<ValidationOverrideRequest> requests)
+        {
+            requests.ToList().ForEach(r => EnsureRequestIsValid(r));
+
+            if (requests.Select(r => new { r.AccountLegalEntityId, r.ULN }).Distinct().Count() != requests.Count())
+            {
+                throw new ArgumentException("Duplicate ValidationOverride entries exist. The combination of AccountLegalEntityId and ULN should be unique.");
+            }
+
+            var response = await _client.PostAsJsonAsync("validation-overrides", requests);            
+
+            response.EnsureSuccessStatusCode();
+        }
+
+        private void EnsureRequestIsValid(ValidationOverrideRequest request)
```

Comment:
I'm thinking this validation logic could be split out into a separate class - separation of concerns